app/views/form_question/show.html.haml

```diff
@@ -3,24 +3,40 @@
 
 %div.container-fluid
   %h1 #{@form_type}
-  %p
-    %font{:color => "Crimson"} #{@error}
 
+  = form_tag form_user_path(:type=> @form_type) do
```

Comment:
Man! This looks too amazing. This block of code just scares me already. We really need to refactor this. I feel like the admin side will need to use some of this logic too.